source/components/molecules/BottomModal/styled.ts

```diff
@@ -14,15 +15,38 @@ interface ModalTopContainerProps {
 }
 const NavigatorContainer = styled.View<ModalTopContainerProps>`
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   justify-content: space-between;
   align-items: center;
-  height: 60px;
   background: ${({ background }) => background};
   border-bottom-color: #eee;
   border-bottom-width: 1px;
-  padding: 0 30px;
   margin-bottom: 16px;
+  border-top-right-radius: 9px;
+  border-top-left-radius: 9px;
+  padding: 0px;
+`;
+
+const HandleBar = styled.View`
+  background-color: #3d3d3d;
+  width: 32px;
+  height: 5px;
+  margin-top: 8px;
+  margin-bottom: 0px;
+  border-radius: 55px;
+`;
+
+const InnerNavigatorContainer = styled.View<ModalTopContainerProps>`
+  display: flex;
+  flex-direction: row;
```

Comment:
flex-direction : row is default in flex container, only needed if the parent has a flex-direction that is not row